documentation/examples/remote_storage/go.mod

```diff
@@ -8,68 +8,73 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/snappy v0.0.4
 	github.com/influxdata/influxdb v1.11.5
-	github.com/prometheus/client_golang v1.18.0
-	github.com/prometheus/common v0.46.0
-	github.com/prometheus/prometheus v0.48.1
+	github.com/prometheus/client_golang v1.19.0
+	github.com/prometheus/common v0.49.0
```

Comment:
I just cut v0.50.0

Reply:
I feel +200 lines from https://github.com/prometheus/common/pull/544 is too much to take last-minute.
It can go in the next release.